README.md

```diff
@@ -51,12 +57,30 @@ This is just JS, so feel free to put Real Code in there.
 
 ## 3. How to invoke webpack
 
+You should install webpack globally:
+
+```shell
+npm install -g webpack
+```
+
 Switch to the directory containing `webpack.config.js` and run:
 
   * `webpack` for building once for development
-  * `webpack -p` for building once for production (minification)
   * `webpack --watch` for continuous incremental build in development (fast!)
   * `webpack -d` to include source maps
+  * `webpack -p` for building once for production (minification)
+
+To run the wepback dev server, run:
+
+```shell
+npm start
+```
+
+Open it in a browser:
+
```

Comment:
Is `it` is referring to the hot reload server? This is going to depend on how you have your app configured.